openeo/internal/jupyter.py

```diff
@@ -0,0 +1,70 @@
+from string import Template
+from collections import Mapping
+import json
+
+# These classes are proxies to visualize openEO responses nicely in Jupyter
+# To show the actual list or dict in Jupyter, use repr() or print()
+
+SCRIPT_URL = 'https://cdn.jsdelivr.net/npm/@openeo/vue-components@2.0.0-beta.1/assets/openeo.js'
+
+class JupyterIntegration():
+
+	# Class instance to store whether the script has been inserted into the DOM
+	# TODO: This fails if a page is reloaded in Jupyter. The variable is still True, but the script doesn't exist in the page any longer.
+	script_sent = False
```

Comment:
you seem to be using tabs here for indentation. The general recommendation/consensus is to use 4 spaces (https://www.python.org/dev/peps/pep-0008/#indentation)

Reply:
Oh, okay. My editor is defaulting to tabs for new files, it seems. Changed.